src/vs/workbench/contrib/terminal/browser/media/shellIntegration-rc.zsh

```diff
@@ -9,6 +9,10 @@ if [ -n "$VSCODE_SHELL_INTEGRATION" ]; then
 	builtin return
 fi
 
+if [[ -f $USER_ZDOTDIR/.zsh_history ]]; then
+	HISTFILE=$USER_ZDOTDIR/.zsh_history
```

Comment:
Does this use `$HOME` if the original `$ZDOTDIR` isn't set?

Reply:
Yes